tests/integration/conftest.py

```diff
@@ -125,6 +128,67 @@ def _integration_test_timeout(request: FixtureRequest) -> None:
     request.node.add_marker(pytest.mark.fail_slow("250ms"))
 
 
+class TimeFreezer:
```

Comment:
As discussed on Slack - think it'd be nice to move this out to a separate file (tests/integration/time_freezer.py)